Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -600,6 +614,28 @@ public final class AppDelegateViewModel: AppDelegateViewModelType, AppDelegateVi
     .flatMap { AppEnvironment.current.pushRegistrationType.hasAuthorizedNotifications() }
     .filter(isTrue)
     .mapConst(0)
+
+    self.configureQualtrics = self.applicationLaunchOptionsProperty.signal.map { _ in
+      return .init(
+        brandId: Secrets.Qualtrics.brandId,
+        zoneId: Secrets.Qualtrics.zoneId,
+        interceptId: QualtricsIntercept.survey.interceptId,
+        stringProperties: [
+          "bundle_id": AppEnvironment.current.mainBundle.bundleIdentifier.coalesceWith("")
```

Comment:
Currently we only sent the `bundle_id` for filtering in the Intercept's configuration. We will likely be adding values here.